src/modules/mod.rs

```diff
@@ -30,6 +31,7 @@ pub fn handle<'a>(module: &str, context: &'a Context) -> Option<Module<'a>> {
         "username" => username::module(context),
         "battery" => battery::module(context),
         "cmd_duration" => cmd_duration::module(context),
+        "job" | "jobs" => jobs::module(context),
```

Comment:
We're soon going to be making all modules only render with a single specific name, so let's to that in advance with this module.
```suggestion
        "jobs" => jobs::module(context),
```